<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWindowCtrlWidget</class>
 <widget class="QWidget" name="CWindowCtrlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CWindowCtrlWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">.QToolButton
{
	background-color:transparent;
	border:none;
	width:50px;
	height:50px;
}

.QToolButton:hover
{
	background-color:rgb(182,182,182);
}

.QToolButton:pressed
{
	background-color:rgb(128,128,128);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_Icon">
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_WindowName">
     <property name="text">
      <string>WindowName</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>512</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QToolButton" name="Btn_Min">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../project/SpeedMonitor.qrc">
       <normaloff>:/resource/min.png</normaloff>:/resource/min.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="Btn_Max">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../project/SpeedMonitor.qrc">
       <normaloff>:/resource/max.png</normaloff>:/resource/max.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="Btn_Close">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../project/SpeedMonitor.qrc">
       <normaloff>:/resource/close.png</normaloff>:/resource/close.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../project/SpeedMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
